portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,54 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Antonio's Portfolio</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1><b>Antonio</b>'s Portfolio</h1>
```

Comment:
Is the `<b>` here just to make the text bold or to bring extra attention to your name? (I know those are pretty closely related and maybe difficult to disentangle, but the [MDN docs on `<b>` might help clear it up](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b).

If you just want your name to be bold, I'd recommend wrapping it in a `<span>` with a class and using the CSS `font-weight` property. Try in general to reserve your HTML document for semantic, structural content, and use CSS for styling (https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns).

Reply:
I thought `<b>` was just designed to make the text bold, but I definitely understand the importance separating concerns!